client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriverUri.java

```diff
@@ -89,8 +89,8 @@ public void testInvalidUrls()
         assertInvalid("jdbc:trino://localhost:8080/blackhole?user=test123", "Connection property 'user' is both in the URL and an argument");
 
         // setting both socks and http proxy
-        assertInvalid("jdbc:trino://localhost:8080?socksProxy=localhost:1080&httpProxy=localhost:8888", "Connection property 'socksProxy' is not allowed");
-        assertInvalid("jdbc:trino://localhost:8080?httpProxy=localhost:8888&socksProxy=localhost:1080", "Connection property 'socksProxy' is not allowed");
+        assertInvalid("jdbc:trino://localhost:8080?socksProxy=localhost:1080&httpProxy=localhost:8888", "Connection property socksProxy cannot be used when httpProxy is set");
+        assertInvalid("jdbc:trino://localhost:8080?httpProxy=localhost:8888&socksProxy=localhost:1080", "Connection property socksProxy cannot be used when httpProxy is set");
```

Comment:
@wendigo this is the new validation I added that we discussed offline. When adding this, I haven't noticed that in other existing error messages, we quote the property name. I wonder which is better:
* Quotes emphasize the property name;
* But all property names use camel case and can't have spaces in them, so they don't otherwise need quoting
* Not using quotes makes the message a bit more terse

Should I add quotes here or remove them elsewhere to make all message consistent?

@mosabua WDYT? Also, these new validation messages are improving the UX quite a bit, so they might be worth of a release note.

Reply:
Please review this new commit https://github.com/trinodb/trino/pull/12587/commits/f762bd1b9c5d13b23ef9daf31319cd6f6fb2c1f0